README.md

```diff
@@ -1,11 +1,9 @@
 odate - Date & Duration Library
 ===============================
 
-Simple date and duration manipulation. Also implement duration printer based
-on string format. Already implemented in opalang [http://opalang.org/]. For
-documentation about the format, see :
-[https://github.com/MLstate/opalang/blob/master/lib/stdlib/core/date/duration.opa]
-(Printing - L472).
+Simple date and duration manipulation. Implements duration printer based
+on string format. Already implemented in [opalang](http://opalang.org/). For
+documentation about the format, see: [Printing - L472](https://github.com/MLstate/opalang/blob/master/lib/stdlib/core/date/duration.opa).
```

Comment:
What about putting the line in the url instead ?
https://github.com/MLstate/opalang/blob/master/lib/stdlib/core/date/duration.opa#L472